packages/flutter/lib/src/services/system_channels.dart

```diff
@@ -224,4 +224,15 @@ class SystemChannels {
     StandardMethodCodec(),
   );
 
+  /// A [MethodChannel] for configuring the Skia graphics library.
+  ///
+  /// The following outgoing methods are defined for this channel (invoked using
+  /// [OptionalMethodChannel.invokeMethod]):
+  ///
+  ///  * `Skia.setResourceCacheMaxBytes`: Set the maximum number of bytes that
+  ///    can be held in the GPU resource cache.
+  static const MethodChannel skia = MethodChannel(
```

Comment:
Maybe name this (and other instances of 'skia`) something like `graphics`, or, alternatively, make this method live on the existing `system` channel.

It's been commented before that Skia is an implementation detail as far as the engine is concerned - and even if we were to say we'd never realistically replace Skia, it's possible Skia would get renamed, etc.

Reply:
@Hixie : any preference for naming of this API?

This message needs to be on a separate channel because it has to be intercepted by the engine (versus the `flutter/system` channel which is forwarded to the embedder).

This particular parameter is specific to the implementation of Skia - if the engine switched to another graphics library then developers' code would need to be updated.

Reply:
`graphics` seems reasonable to me.

Reply:
`skia` is probably also reasonable, since it's Skia-specific, as you say.